config/initializers/filter_parameter_logging.rb

```diff
@@ -40,4 +40,4 @@ def self.filter
   end
 end
 
-Rails.application.config.filter_parameters += [:password, :age, ParameterFiltering.filter]
+Rails.application.config.filter_parameters += [:password, ParameterFiltering.filter]
```

Comment:
This is an unrelated fix that was surfaced by a breaking spec. A full fix will follow up in https://github.com/simpledotorg/simple-server/pull/2519

Reply:
So this PR will start logging ages?

Reply:
Yes. Undid this change in 905a39c, so we can deal with it separately.